legend-engine-xt-relationalStore-sqlserver-pure/src/main/resources/core_relational_sqlserver/relational/tests/dbSpecificTests/customSqlServerTests.pure

```diff
@@ -133,3 +133,18 @@ function <<test.Test>> meta::relational::tests::dbSpecificTests::sqlServer::test
    
    assertPlanGenerationForOptionalParameter(DatabaseType.SqlServer, $expectedPlan);
 }
+
```

Comment:
we don't want to add more tests in this file, as we already db testing framework